proc/process.c

```diff
@@ -634,13 +634,37 @@ static int process_relocate(struct _reloc *reloc, size_t relocsz, char **addr)
 }
 
 
+static int process_relocateAddend(struct _reloc *reloc, size_t relocsz, char **addr, int addend)
```

Comment:
Add addend argument to existing `process_relocate()` and provide zero on armv7m target.
Relocation method differs somewhat, but I think result should be the same.